src/ethereum/vm/stack.py

```diff
@@ -64,3 +64,62 @@ def push(stack: List[U256], value: U256) -> None:
         raise StackOverflowError
 
     return stack.append(value)
+
+
+def push_multiple(stack: List[U256], values: Sequence[U256]) -> None:
+    """
+    Pushes a List of `values` onto `stack`. This should be used over `push`
+    for inserting multiple elements as it is slightly faster than calling
+    `push` for each value.
+
+    Parameters
+    ----------
+    stack : `List[U256]`
+        EVM stack.
+
+    values : `Sequence[U256]`
+        List of items to be pushed onto `stack`.
+
+    Raises
+    ------
+    StackOverflowError
+        If `len(stack)` after inserting the values would be more than `1024`.
+    """
+    if len(stack) + len(values) > 1024:
+        raise StackOverflowError
+
+    stack += values
+
+
+def pop_multiple(stack: List[U256], number_items: int) -> List[U256]:
```

Comment:
I think the readability of multiple individual `pop` calls over one `pop_multiple` trumps performance in this case, unless you've measured a significant difference in speed?